src/viam/robot/__init__.py

```diff
@@ -1,70 +0,0 @@
-"""
-@generated by Viam.
-Do not edit manually!
-"""
-from ..gen.robot.v1.robot_grpc import RobotServiceBase, RobotServiceStub
```

Comment:
won't this change the export structure?

Reply:
yeah, I just want to make sure we didn't intend to make these methods exportable from `viam.robot` (I'll defer to @njooma on that)